scripts/install_environment.sh

```diff
@@ -202,7 +202,7 @@ function additional_configs() {
     sudo chmod ugo+rx "$(dirname "$(pwd)")"
     echo "disabling selinux by setenforce 0"
     sudo setenforce 0 || true
-    sudo chmod 600 ssh_key/key
+    sudo chmod 600 ~/.ssh/id_rsa
```

Comment:
it is needed? maybe better to generate it if doesn't exists ?

Reply:
you're right
take another look